### DISCUSSION TITLE
Squashed commits are rendered in a strange way

### DISCUSSION THREAD

**Discussion by illright** [community/community#37211] [Mobile]
![Screenshot_20221025-154325_GitHub.png](https://user-images.githubusercontent.com/15035286/197776362-391f8c23-6004-406d-8c5f-2d61e1d3de04.png)

It renders `CommitOidAbbreviated` instead of a shortened SHA as I would expect (see screenshot above)

**Comment by Iron-Ham** [comment]
@illright thanks for reporting!
Is this occurring on the iOS or Android app, and which version of the app is this occurring in? Is this happening within the commit list of a repository, commit list of a pull request, or within the timeline of a pull request?

**Reply by spenserblack** [reply]
I've noticed this behavior in the android app, version 1.85.0. I saw it when viewing commits in github/linguist. Sadly I don't remember *exactly* what I did, and I'm struggling to reproduce.

**Reply by illright** [reply]
It's the Android app, yes, latest version. It appeared in place of the "Merge" button after I performed a squash merge of a pull request

**Reply by aconsuegra** [reply]
Hi @illright, thanks for the report! We merged a fix this week, so the new beta that we shipped yesterday should behave properly, and it will be available in our Production build next week.
